modern/public/styles.css

```diff
@@ -10,3 +10,7 @@ canvas {
 .mapboxgl-ctrl {
   margin: 10px;
 }
+
+.maplibregl-popup {
```

Comment:
Does it not adjust automatically based on the content?

Reply:
No. There is an inline style with a max-width of 240px.